packages/crypto/src/multibase.ts

```diff
@@ -24,3 +25,29 @@ export const multibaseToBytes = (mb: string): Uint8Array => {
       throw new Error(`Unsupported multibase: :${mb}`)
   }
 }
+
+export const bytesToMultibase = (
```

Comment:
this package already relies on `multiformats` which has these methods implemented. so we could just export those. not sure why i didn't do so for `multibaseToBytes` 🤔